File: julia/raindrops/raindrops.jl
words = Dict(3=>"Pling",5=>"Plang",7=>"Plong")
getword(num,factor) = mod(num,factor)==0 ? words[factor] :  ""
function raindrops(number)
    output = "$(getword(number,3))$(getword(number,5))$(getword(number,7))"
    length(output) == 0 ? string(number) : output
end

File: csharp/raindrops/Raindrops.cs
public static class Raindrops
{
    public static string Convert(int number)
    {
        var output = string.Empty;

        if (number % 3 == 0)
            output += "Pling";
        if (number % 5 == 0)
            output += "Plang";
        if (number % 7 == 0)
            output += "Plong";

        return output == string.Empty ? number.ToString() : output;
    }
}